src/styles/catalog.scss

```diff
@@ -0,0 +1,36 @@
+.catalog {
+  margin: 50px 40px 72px;
```

Comment:
Hardcoded values should be replaced with SCSS variables for better code maintainability.